tests/pdarray_test.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+import arkouda as ak
+import numpy as np
+import subprocess, os
+
+class PdArrayTest(unittest.TestCase):
+    
+    @classmethod
+    def setUpClass(cls):
+        try: 
```

Comment:
There's some functionality in util/test/util.py for starting/stopping the server and getting host/port information that might be useful here. I think you could do something like:

```python
# probably not the right way to do this..
sys.path.append(os.path.join(os.path.dirname(__file__), '..', 'util', 'test'))
from util import *

...

    def setUpClass(cls):
        start_arkouda_server(get_arkouda_numlocales())

    def setUp(self):
        server_info = get_server_info()
        ak.client.connect(server=server_info.host, port=server_info.port)

...

    @classmethod
    def tearDownClass(cls):
        stop_arkouda_server()
```

That would allow this unit test to work for multi-locale (multi node) configurations. 

There's also a `runClient` wrapper that starts/stops the server and runs client programs that was also meant to be used for unit testing (https://github.com/mhmerrill/arkouda/pull/317), but I'm really not  familiar with python unit testing best practices to know if that was a good approach. I'd be interested in your thoughts on the right way to write/run Arkouda unit tests. The one consideration is that ideally we only want want to start the server once and then run all the unit tests (since queueing times on large clusters can be slow)

Reply:
@ronawho Excellent points, I can integrate in the util methods you mentioned in the setUpClass and tearDownClass methods, which starts up/shuts down the arkouda server for each test suite. Basically each current test file would correspond to one test suite.

Reply:
@ronawho Thanks! the util.test.util.get_arkouda_server works like a champ -> thanks for the tip!